schema.sql

```diff
@@ -1,11 +1,32 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE ANIMALS (
-    id INT,
+    id INT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
     name VARCHAR(100),
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL,
-    species VARCHAR(100)
+    species_id INT,
+    owner_id INT,
+    PRIMARY KEY (id),
+    CONSTRAINT species_key
+        FOREIGN KEY (species_id)
+            REFERENCES species(id),
+    CONSTRAINT owner_key
+        FOREIGN KEY (owner_id)
+            REFERENCES owners(id)
```

Comment:
- [ ] What you did here is good. But we must keep in mind that this is an existing table and we don't really want to directly add new columns like so. Because we will have to drop the table and re-create it with the new columns. Just imagine we have some data already stored, they will be lost right?

For this reason, the SQL keyword `ALERT` can be leveraged to simplify the update.

See example:
```sql
ALTER TABLE animals ADD species_id int REFERENCES species(id);
```

Reply:
Got it @mmsesay. Thanks for the feedback!